Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -221,6 +224,11 @@ private void OnActionRequest(RequestPerformActionEvent ev, EntitySessionEventArg
             return;
 
         DebugTools.Assert(action.AttachedEntity == user);
+        if (action.AttachedEntity == user)
+            action.PreferredEntity = user;
+
+        if (action.Container != null && !HasComp<MindComponent>(action.Container))
+            action.PreferredEntity = action.Container.Value;
```

Comment:
This should either just automatically infer a non-configurable action target and not use a field at all, or it should just use the configurable field without setting it.

i.e., either
```cs
var target = HasComp<MindComponent>() ? user : (action.Container ?? user)
```
or
```cs
var target = action.TargetEntity ?? action.Container ?? user;
```

Reply:
The issue with this was the check on the next part. Because if this isn't done here, then the next method that calls this either does the action on the container or does the action on the entity, and if the action is done on the mind container it will not work properly.

Reply:
I don't understand the problem, you can just pass the correct event target to whatever method raises the event?